src/components/LogItem.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { format } from 'date-fns';
+    import type { UserMessage } from '../models/user.ts';
+    import Waves from './Waves.svelte';
+    export let item: UserMessage;
+
+    const LABELS = {
+        open: 'Bypass: Valve set to open',
+        close: 'Bypass: Valve set to close',
+        bypass: 'Bypass: Bypass mode released',
+        flow: 'Closed valve due to leak detection',
+    };
```

Comment:
This should be an `enum`. Feel free to suppress the Prettier errors regarding camel-casing.
```suggestion
    enum Label {
        open = 'Bypass: Valve set to open',
        close = 'Bypass: Valve set to close',
        bypass = 'Bypass: Bypass mode released',
        flow = 'Closed valve due to leak detection',
    };
```

Reply:
Will consider this.

Reply:
Actually, in hindsight, your way might be cleaner. I'm actually tempted to suggest we just have a dedicated component for each variant where the strings are hard-coded.

Reply:
I'll stick with the current implementation. Let's refactor it if it causes  an issue in the future.